thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/mock/AutoOnboardMockDataSource.java

```diff
@@ -61,16 +61,6 @@ public AutoOnboardMockDataSource(MetadataSourceConfig metadataSourceConfig) {
 
   @Override
   public void run() {
-    if (!this.datasetDAO.findAll().isEmpty()) {
```

Comment:
Are these no longer required?

Reply:
I'm not sure. Let me check with Xiaohui.